hil/cli/switch.py

```diff
@@ -12,18 +14,52 @@ def switch():
 
 @switch.command(name='show')
 @click.argument('switch')
-def node_show(switch):
+@click.option('--jsonout', is_flag=True)
+def switch_show(switch, jsonout):
```

Comment:
this command failed with
```
list index out of range

```

Reply:
Can you give me the json output of the switch that gave you this error?